run_kraken.py

```diff
@@ -246,9 +252,51 @@ def main(cfg):
                                     time.sleep(wait_duration)
                                     cerberus_integration(config)
                                     logging.info("")
+            if litmus_scenarios:
+
+                # Loop to run the scenarios starts here
+                for l_scenario in litmus_scenarios:
+                    try:
+                        for item in l_scenario:
+                            logging.info('item ' + str(item))
+                            runcommand.invoke("kubectl apply -f %s" % item)
+                            with open(item, "r") as f:
+                                yaml_item = list(yaml.safe_load_all(f))[0]
+
+                                logging.info("full load item" + str(type(yaml_item)))
+                            if yaml_item['kind'] == "ChaosEngine":
+                                engine_name = yaml_item['metadata']['name']
+                                namespace = yaml_item['metadata']['namespace']
+                                litmus_namespaces.append(namespace)
+                                experiment_names = yaml_item['spec']['experiments']
+                                for expr in experiment_names:
+                                    expr_name = expr['name']
+                                    experiment_result = common_litmus.check_experiment(engine_name,
+                                                                                       expr_name,
+                                                                                       namespace)
+                                    if experiment_result:
+                                        logging.info("Scenario: %s has been successfully injected!"
+                                                     % item)
+                                    else:
+                                        logging.info("Scenario: %s was not successfully injected!"
+                                                     % item)
+                        for item in l_scenario:
+                            logging.info('item ' + str(item))
+                            runcommand.invoke("kubectl delete -f %s" % item)
+                        logging.info("Waiting for the specified duration: %s" % (wait_duration))
+                        time.sleep(wait_duration)
 
+                        publish_kraken_status(config, failed_post_scenarios)
+                    except Exception as e:
+                        logging.error("Failed to run litmus scenario: %s. Encountered "
+                                      "the following exception: %s" % (item, e))
             iteration += 1
             logging.info("")
+        if litmus_scenarios:
+            for namespace in litmus_namespaces:
+                common_litmus.delete_experiments(namespace)
+            common_litmus.uninstall_litmus()
```

Comment:
can we catch SIGINT (ctrl-c) and clean up Litmus?   I think we're doing that in Cerberus, possibly for slack.   This would help when a) things don't go well with a scenario and it is stuck and b) when running in daemon mode.    We could do this as a follow on PR - not a must to merge this one.

Reply:
Yes definitely think that this would be nice to add. I might do it separate from this PR just to be able to get it in quickly